recipes/r-metalite/bld.bat

```diff
@@ -1,2 +0,0 @@
-"%R%" CMD INSTALL --build . %R_ARGS%
```

Comment:
Please don't remove existing recipes. Those are handled automatically in our CIs. Can you rebase this or close and resubmit if that is easier for you?